conjur/init_controller.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+
+"""
+InitController module
+
+This module is the controller that facilitates all init actions
+required to successfully configure the conjurrc
+"""
+
+# Builtin
+import logging
+import os
+import sys
+
+from urllib.parse import urlparse
+
+import OpenSSL
+
+class InitController:
+    """
+    InitController
+
+    This class represents the Presentation Layer for the init command
+    """
+    DEFAULT_CONFIG_FILE = os.path.expanduser(os.path.join('~', '.conjurrc'))
+
+    force_overwrite = False
+    init_input_obj = None
+    init_logic = None
+
+    def __init__(self, init_input_obj, init_logic, force):
+        self.init_input_obj = init_input_obj
+        self.init_logic = init_logic
+        self.force_overwrite = force
+
+    def call(self):
```

Comment:
"call" is not a good method name. Please change it to indicate what this functionality actually does.

Reply:
Like in Ruby command classes, "class" triggers the execution of multiple functions. IMO it makes the class more organized and intentions clearer. Python has[ __call__](https://www.geeksforgeeks.org/__call__-in-python/) so thought it was known term.
Would `executer` be more clear?

Reply:
Unlike Ruby, this is extremely uncommon to see and it's only used afaik with a `__call__` special function and even that is really just a special constructor helper. The real answer here is that this class should probably be renamed since controller is not a popular Python concept in CLIs and called something like `Configuration`. The method would then be called `load` so that when someone uses the code like this, it's obvious what's going on:
```python
from .config import Configuration

conf = Configuration()
conf.load()
```

Reply:
updated to `load`